recipe/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [[ ${cuda_compiler_version} != "None" ]]; then
+    export TORCH_CUDA_ARCH_LIST="3.5;5.0+PTX"
+    if [[ ${cuda_compiler_version} == 9.0* ]]; then
+        export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;7.0"
+    elif [[ ${cuda_compiler_version} == 9.2* ]]; then
+        export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0"
+    elif [[ ${cuda_compiler_version} == 10.* ]]; then
+        export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5"
+    elif [[ ${cuda_compiler_version} == 11.0* ]]; then
+        export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5;8.0"
+    elif [[ ${cuda_compiler_version} == 11.1 ]]; then
+        export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5;8.0;8.6"
+    elif [[ ${cuda_compiler_version} == 11.2 ]]; then
+        export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5;8.0;8.6"
+    else
+        echo "unsupported cuda version. edit build_pytorch.sh"
+        exit 1
+    fi
+fi
+
+if [[ ${cuda_compiler_version} == "None"  && "$target_platform" == linux-64 ]]; then
+    export FORCE_CUDA="1"
```

Comment:
why are you forcing cuda?

Reply:
But I do see how that causes an issue:
  `OSError: CUDA_HOME environment variable is not set. Please set it to your CUDA install root.` Withe the none cuda builds. 

Ah I just messed up that logic, We only "force cuda" when there is cuda! Oops my bad